src/main/java/org/mockito/verification/VerificationData.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
-package org.mockito.internal.verification.api;
+package org.mockito.verification;
```

Comment:
Making this public is a bit more complicated as it leaks `org.mockito.internal.invocation.InvocationMatcher` class, a simple thing would be to extract an interface from the `InvocationMatcher` class. But I don't want to make public all the API of this class.